lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -28,7 +29,7 @@ class EmbedSanitizer
 
     SUPPORTS_HTTPS = %i[
       ao3 archiveorg eighttracks podfic
-      soundcloud spotify viddersnet viddertube vimeo youtube
+      soundcloud spotify viddersnet viddertube vimeo youtube bilibili
```

Comment:
Just one more comment, can you keep this list sorted as well?

Reply:
done :D